### DISCUSSION TITLE
FE8Nv20210604での魔力分離パッチ導入が上手くいきません

### DISCUSSION THREAD

**Discussion by kimahuresan** [ngmansion/FE8N#103] [Q&A]
FE8Nのv20210604をweb pacherで作成したFE8Nのことです。
ビルダーで魔力分離パッチを導入したらステータス画面がバグって指揮値が表示されません。
EAで魔力分離を導入するとステータス画面やキャラを動かしたときにフリーズします。

v201204ならうまくいったのですが。
最新版で魔力分離パッチをうまく導入する方法はありますか。

**Comment by aera-ngmansion** [comment]
止まる事を確認しました。
魔力パッチの事を考えずに他を変更してしまってた事が原因だと思うので修正します

**Comment by aera-ngmansion** [comment]
@kimahuresan 修正できたと思うので、プルして最新状態にした後にEAでパッチしなおせば直ると思います

**Reply by aera-ngmansion** [reply]
#105

**Reply by aera-ngmansion** [reply]
要らないと思いますが、wikiで全く触れてなかったのでEAでの魔力分離パッチの当て方説明を追加しました
https://github.com/ngmansion/FE8N/wiki/FE8N%E3%81%AE%E4%BD%BF%E3%81%84%E6%96%B9-FE8N%E3%81%AE%E8%A8%AD%E5%AE%9A%E3%82%92%E5%A4%89%E6%9B%B4%E3%81%99%E3%82%8B#%E9%AD%94%E5%8A%9B%E5%88%86%E9%9B%A2%E3%83%91%E3%83%83%E3%83%81%E6%9C%89%E5%8A%B9%E5%8C%96
